src/h5web/providers/models.ts

```diff
@@ -59,7 +59,7 @@ export interface MyHDF5Entity {
   uid: string;
   name: string;
   kind: MyHDF5EntityKind;
-  parents: MyHDF5Group[];
+  parent?: MyHDF5Group;
```

Comment:
Optional so as to allow creating mock entities without parents - i.e. to allow creating child entities before their parents.

Reply:
And also for the root link ?

Reply:
Yes, sorry, that's the main reason ^^ -- but I could have made it optional only on groups.